frontend_lib/src/component/PopinFixed/PopinFixed.styl

```diff
@@ -284,6 +285,23 @@ responsiveColumnStyle()
     & span
       display none
   .wsContentGeneric
+    &__header:has(input)
```

Comment:
instead of using :has(input), prefer adding a class (something like `isEditing`) when in title edition mode and set the specific css rules to the class

Reply:
done